client/setec/store.go

```diff
@@ -434,13 +495,20 @@ func (s *Store) initializeActive(ctx context.Context) error {
 
 	for {
 		var missing int
-		for name, sv := range s.active.m {
-			if sv != nil {
+		for name, cs := range s.active.m {
+			if cs != nil {
 				continue
 			}
 			sv, err := s.client.Get(ctx, name)
 			if err == nil {
-				s.active.m[name] = sv
+				s.active.m[name] = &cachedSecret{
+					Secret:     sv,
+					LastAccess: s.timeNow(),
+					Declared:   true,
+
+					// The only secrets in s.active.m at initialization time are
```

Comment:
Fixme: This is actually not quite true, the only *nil* ones are ones with this property.